packages/wrangler/src/api/startDevWorker/types.ts

```diff
@@ -65,6 +66,22 @@ export interface StartDevWorkerOptions {
 		exclude?: string[];
 	};
 
+	// -- PASSTHROUGH -- FROM OLD CONFIG TO NEW CONFIG (TEMP)
```

Comment:
can you pls expand on this 🙏 ? What's the `old` vs the `new` config?

Reply:
One of the projects that's wrapped up in Start Dev Worker is refactoring Wrangler's internal data structures from a mishmash of props and CfWorkerInit to just use StartDevWorkerOptions. However, since we're doing this in stages it'd be difficult (and inefficient) to shift all of the code over to use the new internal structure immediately, since that would involve converting back and forth in the course of a normal `wrangler dev` session. Instead, for some of these properties we're passing through the old data structure and can clean it up once all the controllers are in place and the data flow is finalised (Config Controller + Bundle Controller are the main ones that need done at this stage)

Reply:
Samuel is correct but I have a commit I didn't push that removes the need for passthrough data structures

Reply:
We're going to address this in a followup